restclient/api_v1.http

```diff
@@ -0,0 +1,8 @@
+### list users
+GET http://localhost:8080/v1/users
+Accept: application/json
+
+### get user by id
+GET http://localhost:8080/v1/users/6796170878378971566
```

Comment:
This http files don't really make sense, right? Each time we run api-server we will regenerate data...

Reply:
This is just my debug tool